raiden/tests/integration/network/transport/rif_comms/test_client.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+
+from raiden.tests.integration.network.transport.rif_comms.node import Node as CommsNode, Config as CommsConfig
+from raiden.tests.integration.network.transport.utils import generate_address
+from transport.rif_comms.utils import notification_to_payload
+
+
+@pytest.fixture()
+@pytest.mark.parametrize("amount_of_nodes")
+def comms_nodes(amount_of_nodes) -> {int, CommsNode}:
+    nodes = {}
+
+    # setup
+    for i in range(1, amount_of_nodes + 1):
+        node = CommsNode(CommsConfig(i))
+        nodes[i] = node
+
+    yield nodes
+
+    # teardown
+    for node in nodes.values():
+        node.stop()
+
+
+@pytest.mark.parametrize("amount_of_nodes", [1])
+def test_locate_own_peer_id(comms_nodes):
+    comms_node = comms_nodes[1]
+
+    # attempt to locate self
+    assert comms_node.client._get_peer_id(comms_node.address) is not ""
+
+
+# FIXME: causes ERR_NO_PEERS_IN_ROUTING_TABLE in comms node although it passes
```

Comment:
documentado en _RIF Comms: pending issues_

Reply:
the test still succeeds. i suspect something funky is going on in the comms node: although it outputs the expected result, it might not have arrived at it through the correct logic.

i.e. "i have no peers in my routing table, therefore i cannot locate this random peer ID you're giving me", should actually be "i do not have this random peer ID in my routing table, so i cannot locate it"